modin/pandas/test/dataframe/test_default.py

```diff
@@ -214,11 +213,39 @@ def test_combine_first():
     df_equals(modin_df1.combine_first(modin_df2), pandas_df1.combine_first(pandas_df2))
 
 
-def test_cov():
-    data = test_data_values[0]
-    modin_result = pd.DataFrame(data).cov()
-    pandas_result = pandas.DataFrame(data).cov()
-    df_equals(modin_result, pandas_result)
+@pytest.mark.parametrize("min_periods", [1, 5, 9])
+def test_corr(min_periods):
+    eval_general(
+        *create_test_dfs(test_data["int_data"]),
+        lambda df: df.corr(min_periods=min_periods),
+    )
+    # AssertionError is arisen because Modin result may slightly differ
+    # from pandas result due to floating pointing arithmetic.
+    try:
+        eval_general(
+            *create_test_dfs(test_data["float_nan_data"]),
+            lambda df: df.corr(min_periods=min_periods),
+        )
+    except AssertionError:
+        pass
```

Comment:
we can pass to `eval_general` custom comparator, that will compare max deviation from pandas result and determine is it small enough to consider that frames are equal
https://github.com/modin-project/modin/blob/f35074c1037b946ba663cabbd0ce9cad0a1567fe/modin/pandas/test/test_groupby.py#L32-L50

Reply:
Passed `modin_df_almost_equals_pandas` as a comparator.